src/helpers/stores/columns.js

```diff
@@ -10,6 +11,20 @@ const UI_COLUMNS_STORAGE_KEY = 'ui-columns';
 const UI_COLUMN_VALUES = Object.values(UI_COLUMN);
 const UI_COLUMN_IDS = UI_COLUMN_VALUES.map((column) => column.id);
 
+DEFAULT_COLUMNS.map((dcolumn) => {
+  UI_COLUMN_VALUES.map((column) => {
+    if (dcolumn.id === column.id) {
+      dcolumn.enabled = config.COLUMNS.includes(column.label)
```

Comment:
Isn't config technically a promise? How are we able to act like it's an object here all of a sudden? 🤔

Reply:
configHelper is an ESM returning config, an object, never a promise
imho you could add it as a module in package.json instead of using await since Node v13?, the result would be the same